source/whats-new.txt

```diff
@@ -20,6 +21,47 @@ Learn what's new in:
 * :ref:`Version 4.1 <version-4.1>`
 * :ref:`Version 4.0 <version-4.0>`
 
+.. _version-4.7:
+
+What's New in 4.7
+-----------------
+
+New features of the 4.7 driver release include:
+
+- Improved change stream support
```

Comment:
Issue: style guide "Introduce a list with a sentence, and end the sentence with a colon."
Suggestion: "Added the following change stream support improvements:"